webpack.config.js

```diff
@@ -53,11 +51,6 @@ module.exports = {
 			},
 		],
 	},
-	node: {
-		fs: 'empty',
-		child_process: 'empty',
-		__dirname: false,
-	},
```

Comment:
[Thanks to Ben](https://github.com/getflywheel/local-addon-instant-reload/pull/21) for this.